app/views/steps/conviction/conviction_subtype/edit.html.erb

```diff
@@ -1,23 +1,15 @@
 <% title t('.page_title') %>
+<% step_header %>
 
-<div class="govuk-width-container">
-  <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
 
-  <main id="main-content" class="govuk-main-wrapper">
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_collection_radio_buttons :conviction_subtype, @form_object.values, :value, nil,
+            legend: { text: f.i18n_legend } %>
```

Comment:
This requires also hints (anything that before had `radio_hint: true`) using the `i18n_hint`
Otherwise hints will not show.

Reply:
No worries, I was confused as well when I did it